### PR TITLE
planify: 4.4 -> 4.5.2

### PR DESCRIPTION
**Pull request by jmarmstrong1207**
## Description of changes
Update planify to 4.5. Requires libsecret and new library gxml.

Requires #292024 to be merged first.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/office/planify/default.nix [hunk 1]
```diff
@@ -1,4 +1,5 @@
-{ stdenv
+{ callPackage
+, stdenv
```
Comment (bobby285271) [inline pkgs/applications/office/planify/default.nix:2]: Please revert this changes here :bow:

File: pkgs/applications/office/planify/default.nix [hunk 2]
```diff
@@ -12,26 +13,27 @@
...
 , libgee
 , libical
 , libportal-gtk4
+, libsecret
 , libsoup_3
 , pantheon
 , sqlite
 , webkitgtk_6_0
 }:
-
```
Comment (bobby285271) [inline pkgs/applications/office/planify/default.nix:25]: And also revert this change

**Comment by bobby285271** [comment]
@ofborg eval
@ofborg build planify

**Comment by bobby285271** [comment]
Let me know if this can be undrafted.

(And [4.5.2](https://github.com/alainm23/planify/compare/4.5...4.5.2) released).

**Comment by jmarmstrong1207** [comment]
> Please let me know if this can be undrafted.
> 
> (And [4.5.2](https://github.com/alainm23/planify/compare/4.5...4.5.2) released).

Oh yeah, thanks for the reminder, sorry about that.

I'll also update it later, but if you're reviewing its fine to do it yourself if you want it to be committed quicker
